src/manage/getAccessRequestFromRedirectUrl.test.ts

```diff
@@ -22,6 +22,8 @@
 // eslint-disable-next-line no-shadow
 import { describe, it, jest, expect } from "@jest/globals";
 import { getVerifiableCredential } from "@inrupt/solid-client-vc";
+// eslint-disable-next-line no-shadow
+import { btoa } from "node:buffer";
```

Comment:
Why not replace this with base64url from jose too?

Reply:
Could do. This was just simpler.